dev-support/jenkins-scripts/generate-hbase-website.sh

```diff
@@ -191,6 +191,10 @@ else
   exit $status
 fi
 
+# Replace MathJax CDN URI with local one in book.html
+echo "Replace MathJax URI"
```

Comment:
Please add a link to the upstream issue so that we can eventually unwind this.

Reply:
Thanks, good idea. Added a comment and the links.